### ISSUE TITLE
Use .any.js for xhr tests

### ISSUE THREAD

**Issue by zcorpan** [web-platform-tests/wpt#11588]
Tests in xhr/ seem like good candidates for [.any.js](https://web-platform-tests.org/writing-tests/testharness.html#multi-global-tests). (Noticed while working on #11571.)

**Comment by priyapulakhandam** [comment]
Hey @zcorpan,i really want to try and contribute towards this.

**Comment by Jeevita898** [comment]
hey @zcorpan  .. can u assign this issue to me

**Comment by zcorpan** [comment]
Go ahead and submit a PR, no need to assign. Thanks!

**Comment by Boopana-M** [comment]
I want to work on this

**Comment by annevk** [comment]
I don't think we need to track this and it's also not immediately obvious we want to double or triple or quadruple the number of tests we run with likely the same outcomes.
